packages/backend/mongo/sessionSchema.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+
+const sessionSchema = new mongoose.Schema({
+  teamName: String,
+  roomName: String
+});
+
+Session = mongoose.models['tests'] || mongoose.model('tests', sessionSchema);

```

Comment:
what are tests model?

Reply:
It's the temporary name of the model for the room declaration schema. Once I'll have more data to push to mongoDB i'll rename it.